### PR TITLE
added CartServiceImplTest

### PR DESCRIPTION
**Pull request by aliaksharbachou**

### CODE REVIEW

File: src/test/java/internship/mbicycle/storify/unit/CartServiceImplTest.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+import static org.mockito.BDDMockito.given;
+import static org.mockito.BDDMockito.then;
+import static org.mockito.Mockito.only;
+
+@ExtendWith(MockitoExtension.class)
+public class CartServiceImplTest {
+    private static final String EMAIL = "Igor";
+    private static final String ROLE = "role";
+    private static final String PASSWORD = "password";
+    private static Authentication auth;
```
Comment (Estrin-MBicycle) [inline src/test/java/internship/mbicycle/storify/unit/CartServiceImplTest.java:30]: why do we need global variable?

**Review by Estrin-MBicycle — CHANGES_REQUESTED** [review]

File: src/test/java/internship/mbicycle/storify/unit/CartServiceImplTest.java (hunk 1 above)
Comment (aliaksharbachou) [reply src/test/java/internship/mbicycle/storify/unit/CartServiceImplTest.java:30]: I changed

File: src/test/java/internship/mbicycle/storify/unit/CartServiceImplTest.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    private StorifyUserService storifyUserService;
+
+    @Mock
+    private CartRepository cartRepository;
+
+    @InjectMocks
+    private CartServiceImpl cartService;
+
+    @Test
+    void testFindByCart() {
```
Comment (andrey11m) [inline src/test/java/internship/mbicycle/storify/unit/CartServiceImplTest.java:56]: Why the name is "findByCart"?

File: src/test/java/internship/mbicycle/storify/unit/CartServiceImplTest.java [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        given(cartRepository.save(cart)).willReturn(cart);
+        cartService.addProduct(cart, 20L, 3);
+        cartService.changeProductByCount(cart, 20L, 5);
+        Integer expectedCount = 5;
+        Integer actualCount = cart.getProductsMap().get(20L);
+        assertEquals(expectedCount, actualCount);
+    }
+
+    private static StorifyUser getUser() {
+        StorifyUser storifyUser = new StorifyUser();
```
Comment (andrey11m) [inline src/test/java/internship/mbicycle/storify/unit/CartServiceImplTest.java:84]: Why you didn't use builder?

File: src/test/java/internship/mbicycle/storify/unit/CartServiceImplTest.java (hunk 2 above)
Comment (aliaksharbachou) [reply src/test/java/internship/mbicycle/storify/unit/CartServiceImplTest.java:56]: changed

File: src/test/java/internship/mbicycle/storify/unit/CartServiceImplTest.java (hunk 3 above)
Comment (aliaksharbachou) [reply src/test/java/internship/mbicycle/storify/unit/CartServiceImplTest.java:84]: because the builder is not suitable here

**Pull request merged**